tests/modelling/test_engine.py

```diff
@@ -430,3 +430,43 @@ def test_drawdown(self):
                 columns=assets,
             ),
         )
+
+
+class MultiColumnLoaderTestCase(TestCase):
+    def setUp(self):
+        self.assets = [1, 2, 3]
+        self.dates = date_range('2014-01-01', '2014-02-01', freq='D', tz='UTC')
+
+        asset_info = make_simple_asset_info(
+            self.assets,
+            start_date=self.dates[0],
+            end_date=self.dates[-1],
+        )
+        self.asset_finder = AssetFinder(asset_info)
+
+    def test_engine_with_multicolumn_loader(self):
+        open_, close = USEquityPricing.open, USEquityPricing.close
+
+        loader = MultiColumnLoader({
+            open_: ConstantLoader(dates=self.dates,
+                                  assets=self.assets,
+                                  constants={open_: 1}),
+            close: ConstantLoader(dates=self.dates,
+                                  assets=self.assets,
+                                  constants={close: 2})
+        })
+
+        engine = SimpleFFCEngine(loader, self.dates, self.asset_finder)
+
+        factor = RollingSumDifference()
+
+        result = engine.factor_matrix({'f': factor},
+                                      self.dates[2],
+                                      self.dates[-1])
+        self.assertIsNotNone(result)
```

Comment:
> I thought it would be nice to have the test fail with "unexpectedly None" as opposed to "TypeError: argument of type 'NoneType' is not iterable".

The sentiment here makes sense to me, but on the other hand, it seems like this implies that we should put `self.assertIsNotNone` as the first check before every function we test anywhere, which seems incorrect to me.  I don't care super strongly either way though.